identity-apps-core/apps/authentication-portal/src/main/webapp/includes/product-footer.jsp

```diff
@@ -54,17 +54,7 @@
 %>
 
 <%-- Cookie Consent Banner --%>
-<%
-    if (config.getServletContext().getResource("extensions/cookie-consent-banner.jsp") != null) {
-%>
-        <jsp:include page="extensions/cookie-consent-banner.jsp"/>
-<%
-    } else if (config.getServletContext().getResource("includes/cookie-consent-banner.jsp") != null) {
-%>
-        <jsp:include page="includes/cookie-consent-banner.jsp"/>
-<%
-    }
-%>
+<jsp:include page="cookie-consent-banner.jsp"/>
```

Comment:
I think we need to keep this extension point so that we can keep the cloud specific cookie banner configuration separately.